trunk/src/protocol/srs_protocol_raw_avc.cpp

```diff
@@ -400,7 +400,16 @@ srs_error_t SrsRawHEVCStream::mux_sequence_header(std::string vps, std::string s
     //      sequenceParameterSetNALUnit
 
     // use simple mode: nalu size + nalu data
-    int nb_packet = 23 + 5 + (int)vps.length() + 5 + (int)sps.length() + 5 + (int)pps.length();
+    /**/
+	int nPPSSize = 0;
```

Comment:
This code should use four spaces instead of tabs, and it would be great to maintain consistent code style and variable naming. Thank you very much!
```c++
    // pps size
    int pps_size = 0;
    for (std::vector<std::string>::iterator it = pps->begin(); it != pps->end(); it++) {
        pps_size += 2 + it->length();
    }

    int nb_packet = 23 + 5 + (int)vps.length() + 5 + (int)sps.length() + 5 + pps_size - 2;
    char *packet = new char[nb_packet];
```